### PR TITLE
Fix/cli/525/git setup remote already exists

### LINKED ISSUE #525: [Git-Setup] This command should not fail if the remote already exist
If the remote already exist, it should update it instead of just complaining.

### PR DESCRIPTION
**Pull request by cyrille-sg**
Offer a option --force to the command git-setup to allow the user to override the current remote handle with a new remote url

Fix #525 

User
- New flag "--force" on "git-seup"
- Suggestion message to use force when already remote handle already exist
Tech
- Propose change in errors management (using utils.Wrap instead of in place errgo.Notef)

### CODE REVIEW

File: cmd/git.go [hunk 1]
```diff
@@ -29,7 +32,8 @@ var (
 			}
 
 			err := git.Setup(appdetect.CurrentApp(c), git.SetupParams{
-				RemoteName: c.String("r"),
+				RemoteName:     c.String("r"),
```
Comment (Soulou) [inline cmd/git.go]: Please use the long flag name for a better readability

File: git/setup.go [hunk 2]
```diff
@@ -24,14 +26,66 @@ func Setup(appName string, params SetupParams) error {
...
+func createRemoteInRepository(repository *git.Repository, remoteName string, url string) error {
+	_, err := repository.CreateRemote(&gitconfig.RemoteConfig{
+		Name: remoteName,
+		URLs: []string{url},
+	})
+	if err != nil {
+		errWrapped := utils.WrapError(err, "fail to create the Git remote")
+		if err == git.ErrRemoteExists {
+			errWrapped = utils.WrapError(err, "fail to create the Git remote"+
+				" (maybe you want use the force option to override the remote url)")
```
Comment (Soulou) [inline git/setup.go]:
```
Fail to configure git repository, "%v" remote already exists. (Use --force option to override)
```

**Comment by cyrille-sg** [comment]
Screenshot from 2020-03-11 11-39-07

**Comment by Soulou** [comment]
As spoken orally, the error displayed to the user is not right (repetition), to make it cleaner, it requires some changes in way we're handling error and probably to modify https://github.com/Scalingo/go-scalingo/errors

**Comment by Soulou** [comment]
After disucssion, we accept this version, better error management will be handled later

**Pull request merged**